delta/migration/src/test/scala/ch/epfl/bluebrain/nexus/migration/v1_4/SourceSanitizerSpec.scala

```diff
@@ -0,0 +1,36 @@
+package ch.epfl.bluebrain.nexus.migration.v1_4
+
+import ch.epfl.bluebrain.nexus.testkit.CirceLiteral
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+
+class SourceSanitizerSpec extends AnyWordSpecLike with Matchers with CirceLiteral {
+
+  "Replacing old contexts" should {
+
+    "work in an array" in {
+      val original =
+        json"""{"@context" : ["https://bbp.neuroshapes.org","https://bluebrain.github.io/nexus/contexts/resource.json"]}"""
```

Comment:
this is very specific to BBP, is there a way to handle this problem without fixed contexts?

Reply:
We can handle it via some configuration but I want to test things and have a better idea first